### ISSUE TITLE
Support session config

### ISSUE THREAD

**Issue by pi0** [unjs/h3#693]
H3 supports session using `useSession(event, config)` however it's usage is not initiative because each time we need to manually handle the `config` passing.

I am thinking to support `useSession(event, name: string)` overload that tries to read session config from h3 app configuration.

**Comment by ultimateshadsform** [comment]
I agree it would be nice to be able to have some global config and grab the secret/password using `runtimeConfig`.

Though right now I just wrap the useSession in it's own function so I just use that wrapped function everywhere that grabs the config from a global thing.
